cpp/ql/src/semmle/code/cpp/models/implementations/Pure.qll

```diff
@@ -7,7 +7,7 @@ import semmle.code.cpp.models.interfaces.SideEffect
 private class PureStrFunction extends AliasFunction, ArrayFunction, TaintFunction,
   SideEffectFunction {
   PureStrFunction() {
-    hasGlobalOrStdName([
+    hasGlobalOrStdishName([
```

Comment:
I trust you've checked through the BSL docs / headers and all of these functions at least appear to match their standard library counterparts?  (in my PRs I've generally just checked that the parameters are what I expect, because it's a lot of work to dig any deeper than that)

Reply:
In all the cases for the C standard functions, BSL just imports the C++ counterparts from `std` into `bsl` - so they are 100% identical.